modules/apps/29-fee/keeper/msg_server.go

```diff
@@ -36,13 +36,15 @@ func (k Keeper) RegisterPayee(goCtx context.Context, msg *types.MsgRegisterPayee
 	return &types.MsgRegisterPayeeResponse{}, nil
 }
 
-// RegisterCounterpartyAddress is called by the relayer on each channelEnd and allows them to specify their counterparty address before relaying
-// This ensures they will be properly compensated for forward relaying on the source chain since the destination chain must send back relayer's source address (counterparty address) in acknowledgement
-// This function may be called more than once by relayers, in which case, the previous counterparty address will be overwritten by the new counterparty address
-func (k Keeper) RegisterCounterpartyAddress(goCtx context.Context, msg *types.MsgRegisterCounterpartyAddress) (*types.MsgRegisterCounterpartyAddressResponse, error) {
+// RegisterCounterpartyPayee defines a rpc handler method for MsgRegisterCounterpartyPayee
+// RegisterCounterpartyPayee is called by the relayer on each channelEnd and allows them to specify their
+// counterparty address before relaying. This ensures they will be properly compensated for forward relaying since
```

Comment:
```suggestion
// RegisterCounterpartyPayee is called by the relayer on each channelEnd and allows them to specify the
// counterparty payee address before relaying. This ensures they will be properly compensated for forward relaying since
```

Reply:
I'll update these docs both here and in the proto files as they're replicated and push the changes 👍